.github/workflows/docker.yml

```diff
@@ -0,0 +1,55 @@
+# don't use this until we start using versions and docker
```

Comment:
Should we instead delete the file and open issue what actions need to be done before we can add it back?